core/core/test/serverspec/spec/master/master_spec.rb

```diff
@@ -1,28 +1,166 @@
 require 'spec_helper'
+require 'base64'
 
-describe service('kubelet') do
-  it { should be_enabled }
-  it { should be_running }
+
+describe 'Check if kubelet service is up' do
+  describe service('kubelet') do
+    it { should be_enabled }
+    it { should be_running }
+  end
+end
+
```

Comment:
Please add test for Kubelet that checks for /healtz enpoint response
` curl --insecure -I -m 2 https://127.0.0.1:10250/healthz`
401 (Unauthorized) response is fine